### PR TITLE
fix tiktok downloads

### LINKED ISSUE #396: TikTok API is unreachable
**bug description**
tiktok downloading not works.

**links**
[if applicable, add links that cause the issue. more = better](https://www.tiktok.com/@ahsaneditz100/video/7348445366066384159)

### PR DESCRIPTION
**Pull request by wukko**
closes #396 and #294

remake of #401

### CODE REVIEW

File: src/localization/languages/en.json
Comment (wukko) [inline src/localization/languages/en.json:1]: that's a period

File: src/modules/processing/services/tiktok.js [hunk 1]
```diff
@@ -48,24 +27,25 @@ export default async function(obj) {
...
+
+    let detail = await fetch(`${apiURL.href}&${deviceInfo}`, {
         headers: {
-            "user-agent": config[obj.host].userAgent
+            "user-agent": apiUserAgent
         }
     }).then((r) => { return r.json() }).catch(() => { return false });
 
-    detail = selector(detail, obj.host, postId);
+    detail = detail?.aweme_list?.filter(v => v.aweme_id === postId)[0];
```
Comment (dumbmoron) [inline src/modules/processing/services/tiktok.js:42]: Screenshot 2024-03-29 at 1 37 15

Comment (wukko) [reply src/modules/processing/services/tiktok.js:42]: 😭

**Review by dumbmoron — APPROVED** [review]
llvm 
image

**Pull request merged**